neofetch

```diff
@@ -2772,6 +2781,14 @@ dynamicprompt() {
     # Get cursor position
     info_height="$(IFS=';' builtin read -srdR -t 1 -d c -p $'\033[6n\033[c' ROW COL; printf "%s" "${ROW#*[}")"
 
+    # On Windows 10 subsystem for Linux, the default terminal does not
+    # produce any integer output, so a workaround is used.
+    if [[ $info_height =~ ^-?[0-9]+$ ]]; then
```

Comment:
What's the output? What you could do is use a case statement instead, so something like this:

```sh
case "$info_height" in
    "-"?[0-9]*) ;;
    *) info_height="0" ;;
esac
```